templates/quiz/create.html

```diff
@@ -21,9 +21,9 @@ <h3>Please remember:</h3>
 
         <p><label for="id_result">Result:</label> {{ form.result }} {{ form.result.errors }}</p>
 
-        <p><label for="id_answer">The output of this program is:</label> {{ form.answer }} {{ form.answer.errors }}<span class="help_text">Leave blank for questions with errors / undefined behavior</span></p>
+        <p><label for="id_answer">The output of this program is:</label> {{ form.answer }} {{ form.answer.errors }}<span class="help_text">Leave blank for questions with errors / undefined behaviour</span></p>
```

Comment:
I, too, prefer British English, but since the standard is written in American English and we quote it quite a lot, let's stick to American English for the site. (There's probably a good mix currently.)

Reply:
Yeah, I wouldn't have changed this line (since that's not a typo) if I hadn't seen another inconsistent usage on the same page.

Now both usages are in American English.

Reply:
I see, thanks!